embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/AutoStartInstrumentationHook.java

```diff
@@ -0,0 +1,28 @@
+package io.embrace.android.embracesdk;
+
+import androidx.annotation.NonNull;
+
+import io.embrace.android.embracesdk.annotation.InternalApi;
+import io.embrace.android.embracesdk.internal.EmbraceInternalApi;
+
+/**
+ * @hide
+ */
+@InternalApi
+public final class AutoStartInstrumentationHook {
+
+    private AutoStartInstrumentationHook() {
+    }
+
+    public static void _preOnCreate(android.app.Application application) {
+        try {
+            Embrace.getInstance().start(application);
+        } catch (Exception exception) {
+            logError(exception);
+        }
+    }
+
+    private static void logError(@NonNull Throwable throwable) {
```

Comment:
This won't do anything if the SDK isn't started so IMO could be deleted, and we just swallow any unexpected exception